modules/engage-paella-player/src/main/resources/OSGI-INF/paella-config-file.xml

```diff
@@ -1,6 +1,6 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <scr:component xmlns:scr="http://www.osgi.org/xmlns/scr/v1.1.0"
- name="org.opencastproject.engage.paella.PaellaConfigRest" immediate="true" activate="activate" deactivate="deactivate">
```

Comment:
But PaellaConfigRest does have an activate method... why have you removed "activate"?

Reply:
Because it's `activate` and if there is an `activate` method you do not need to name it here explicitly. It will be picked up automatically.

Reply:
Ok, thanks for the clarifications.